src/pages/Dashboard/Dashboard.tsx

```diff
@@ -89,100 +69,78 @@ export const Dashboard = memo(() => {
     walletDispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
   }, [isLoadingLocalWallet, isConnected, walletDispatch])
 
-  const handleSlideIndexChange = useCallback(
-    (index: number) => {
-      switch (index) {
-        case MobileTab.Overview:
-          navigate('')
-          break
-        case MobileTab.Earn:
-          navigate('earn')
-          break
-        case MobileTab.Activity:
-          navigate('activity')
-          break
-        default:
-          break
-      }
-
-      setSlideIndex(index)
-    },
-    [navigate],
-  )
```

Comment:
No need for that either, this was only used for mobile (web, really) and *now* works perfectly fine without:
https://jam.dev/c/86fd27cc-5acf-4af6-b91f-2c079f402dce

See develop for comparison:

https://jam.dev/c/38a84c17-2d33-4c52-9b09-5f627ba0f2e7

The only reason we had that was for the routing to match the subroute, which isn't required anymore since Watchlist isn't a route but just a tab of home.